packages/communication-react/review/communication-react.api.md

```diff
@@ -1426,6 +1426,11 @@ export interface MeetingCallControlOptions extends Pick<CallControlOptions, 'cam
 // @beta
 export const MeetingComposite: (props: MeetingCompositeProps) => JSX.Element;
 
+// @beta
+export type MeetingCompositeOptions = {
+    meetingCallControlOptions?: boolean | MeetingCallControlOptions;
```

Comment:
I think the prefix `meeting` and suffix `options` here is verbose.
I suggest naming it simply `callControls`

Reply:
Sounds good will make it so!